aten/src/ATen/native/native_functions.yaml

```diff
@@ -1789,6 +1789,8 @@
 
 - func: new_empty_strided(Tensor self, int[] size, int[] stride, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
   variants: method
+  dispatch:
```

Comment:
nit: add a comment above this one explaining that we do not want to make new_empty a composite that calls into this one as the strided version is significantly harder to implement for backends and it is easier for them to work with new_empty.